code/modules/mob/living/simple_animal/hostile/alien.dm

```diff
@@ -78,16 +91,110 @@
 	move_to_delay = 4
 	maxHealth = 400
 	health = 400
+	base_attack_cooldown = 12
+	ai_holder = /datum/ai_holder/simple_animal/urist_humanoid/alien/queen/empress
 	natural_armor = list(
-		melee = ARMOR_MELEE_KNIVES,
+		melee = ARMOR_MELEE_RESISTANT,
 		laser	= ARMOR_LASER_HANDGUNS,
-		energy	= ARMOR_ENERGY_SMALL
+		energy	= ARMOR_ENERGY_SMALL,
+		ballistic = ARMOR_BALLISTIC_MINOR
 		)
 
 /obj/item/projectile/neurotox
-	damage = 30
+	damage = 25
+	damage_type = DAMAGE_BURN
```

Comment:
I guess we can avoid the neurotox-firestarting issue by overriding the proc that causes it. There is a risk that this will drift from the Bay implementation though.
```
/obj/item/projectile/neurotox/on_impact(atom/A)
	impact_effect()		// generate impact effect
        // cut out the firestarting bits from Bay that were here
        return
```